src/store/metadata/stub/content/editions/versions/beis_2005-2019_la_carbon_emission.json

```diff
@@ -0,0 +1,102 @@
+{
```

Comment:
same deal with file naming as the other one, literally the same name in fact, just at the /versions level.